portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,27 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import java.io.PrintWriter;
+import java.lang.Math.*;
```

Comment:
Only import what you use. Don't use *

Reply:
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Fixed! Actually, the math library is a single package, so I had just to remove the asterisk and it worked! My bad! :)